modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

```diff
@@ -2873,6 +3054,59 @@ Total database accesses: 318, total allocated memory: 208
 
 ======
 
+[[query-plan-nullify-metadata]]
+== Nullify Metadata
+// NullifyMetadata
+_This feature was introduced in Neo4j 5.9._
+NullifyMetadata is responsible for cleaning up state associated with Repeat(Trail), it is only planned directly after Repeat(Trail).
```

Comment:
Im not sure if my suggestion to add a "the" before "state" is correct here. In any case, can we explain briefly what the "state" refers to here?

```suggestion
_This feature was introduced in Neo4j 5.9._

`NullifyMetadata` is responsible for cleaning up the state associated with `Repeat(Trail)`.
It is only planned directly after `Repeat(Trail)`.
```